test/Flora/PackageSpec.hs

```diff
@@ -74,56 +77,37 @@ testFetchGHCPrimDependents = do
     (Set.fromList . fmap (view #name) $ Vector.toList result)
 
 testThatBaseisInPreludeCategory :: Fixtures -> TestM ()
-testThatBaseisInPreludeCategory Fixtures{hackageUser} = do
-  liftDB $ importPackage (hackageUser ^. #userId) (PackageName "base") "./test/fixtures/Cabal/"
+testThatBaseisInPreludeCategory fixtures = do
+  let cabalPath = "./test/fixtures/Cabal/base.cabal"
+  liftDB $ importFile (fixtures ^. #hackageUser % #userId) cabalPath
   result <- liftDB $ Query.getPackagesFromCategorySlug "prelude"
   assertEqual (Set.fromList [PackageName "base"]) (Set.fromList $ V.toList $ fmap (view #name) result)
 
 testThatSemigroupsIsInMathematicsAndDataStructures :: Fixtures -> TestM ()
-testThatSemigroupsIsInMathematicsAndDataStructures Fixtures{hackageUser} = do
-  liftDB $ importPackage (hackageUser ^. #userId) (PackageName "semigroups") "./test/fixtures/Cabal/"
+testThatSemigroupsIsInMathematicsAndDataStructures fixtures = do
+  let cabalPath = "./test/fixtures/Cabal/semigroups.cabal"
+  liftDB $ importFile (fixtures ^. #hackageUser % #userId) cabalPath
   Just semigroups <- liftDB $ Query.getPackageByNamespaceAndName (Namespace "hackage") (PackageName "semigroups")
   result <- liftDB $ Query.getPackageCategories (semigroups ^. #packageId)
   assertEqual (Set.fromList ["data-structures", "maths"]) (Set.fromList $ slug <$> V.toList result)
 
 testCorrectNumberInHaskellNamespace :: Fixtures -> TestM ()
-testCorrectNumberInHaskellNamespace Fixtures{hackageUser} = do
-  liftDB $
-    importCabal
-      (hackageUser ^. #userId)
-      (PackageName "Cabal")
-      "./test/fixtures/Cabal/Cabal.cabal"
-      "./test/fixtures/Cabal/"
+testCorrectNumberInHaskellNamespace fixtures = do
+  importAllPackages fixtures
   results <- liftDB $ Query.getPackagesByNamespace (Namespace "haskell")
   assertEqual 21 (Vector.length results)
```

Comment:
This test was wrong, could you perhaps change this number from hardcoded 21 to `Set.size coreLibraries`?